src/pages/workspace/WorkspacesListPage.tsx

```diff
@@ -1,76 +1,57 @@
-import PropTypes from 'prop-types';
+import _ from 'lodash';
```

Comment:
Let's remove any lodash and underscore usage if possible

Reply:
I think we cannot remove the lodash in this file